cantor-archive/src/main/java/com/salesforce/cantor/archive/FileArchiver.java

```diff
@@ -0,0 +1,13 @@
+package com.salesforce.cantor.archive;
+
+import com.salesforce.cantor.misc.archivable.Archiver;
+import com.salesforce.cantor.misc.archivable.EventsArchiver;
+
+import java.nio.file.Path;
+
+public class FileArchiver implements Archiver<Path> {
```

Comment:
@AlexanderKouthoofd `Archiver<Path>`? does this compile?

Reply:
Oh I thought I cleaned this up. I removed this generic on implementation branch. I'll remove it here.

Reply:
The `archiver` module has been completely rewritten. Some of my experiments slipped in here when I was breaking up the PR. I added all the new stuff in as no-ops, so you can get a better picture.